src/markdown-pages/docs/managers/mq-manager.md

```diff
@@ -0,0 +1,227 @@
+
+The following snippet shows the code required to read a message from an existing IBM MQ queue:
+
+```java
+Message m = queue.getMessage();
+String response = m.getBody(String.class);
+```
+</details>
+
+<details><summary>Clear messages from an IBM MQ queue</summary>
+
+The following snippet shows the code required to clear messages from an IBM MQ queue:
+
+```java
+queue.clearQueue();
+```
+</details>
+
+## Configuration Properties
+
+The following are properties used to configure the MQ Manager.
+ 
+<details>
+<summary>Queue manager channel CPS Property</summary>
+
+| Property: | Queue manager channel CPS Property |
+| --------------------------------------- | :------------------------------------- |
+| Name: | mq.server.[instanceid].channel |
+| Description: | The channel for the specified instance |
+| Required:  | Yes |
+| Default value: | None |
+| Valid values: |  |
+| Examples: | <code>mq.server.[instanceid].channel=DEV.APP.SVRCONN</code> |
+
+</details>
+ 
+<details>
+<summary>Queue manager credentials CPS Property</summary>
+
+| Property: | Queue manager credentials CPS Property |
+| --------------------------------------- | :------------------------------------- |
+| Name: | mq.server.[instanceid].credentials.id |
+| Description: | The credentials for the queue Manager |
+| Required:  | Yes |
+| Default value: | None |
+| Valid values: |  |
+| Examples: | <code>mq.server.[instanceid].credentials.id=CRED1</code> |
+
+</details>
+
+<details>
+<summary>Queue manager host CPS Property</summary>
+
+| Property: | Queue manager host CPS Property |
+| --------------------------------------- | :------------------------------------- |
+| Name: | mq.server.[instanceid].host |
+| Description: | The host for the specified instance |
+| Required:  | Yes |
+| Default value: | None |
+| Valid values: |  |
+| Examples: | <code>mq.server.[instanceid].host=127.0.0.1</code> |
+
+</details>
+ 
+<details>
+<summary>Queue manager name CPS Property</summary>
+
+| Property: | Queue manager name CPS Property |
+| --------------------------------------- | :------------------------------------- |
+| Name: | mq.server.[instanceid].name |
+| Description: | The queue manager name for the specified instance |
+| Required:  | Yes |
+| Default value: | None |
+| Valid values: |  |
+| Examples: | <code>mq.server.[instanceid].name=QM1</code> |
+
+</details>
+
+<details>
+<summary>Queue manager port CPS Property</summary>
+
+| Property: | Queue manager port CPS Property |
+| --------------------------------------- | :------------------------------------- |
+| Name: | mq.server.[instanceid].port |
+| Description: | The queue manager port for the specified instance |
+| Required:  | Yes |
+| Default value: | None |
+| Valid values: |  |
+| Examples: | <code>mq.server.[instanceid].port=1414</code> |
+
+</details>
+
+<details>
+<summary>Instance ID for the tag CPS Property</summary>
+
+| Property: | Instance ID for the tag CPS Property |
```

Comment:
This should be the first property mentioned and we should show how it relates to the tag attributes in the @QueueManager annotation

Reply:
updated